src/Advertising.js

```diff
@@ -11,16 +11,13 @@ export default class Advertising {
     this.customEventCallbacks = {};
     this.customEventHandlers = {};
     this.queue = [];
-    this.isPrebidUsed = typeof window.pbjs !== 'undefined';
```

Comment:
this is what caused the crash when using react-advertising on the server: we can't assume there is a `window` object when the class is constructed. I've fixed it by moving the check for Prebid being used to the init method, which will only run client-side